src/Map.jsx

```diff
@@ -1,17 +1,55 @@
-import React, { useEffect, useRef, useState } from "react";
-import { MapContainer, TileLayer } from "react-leaflet";
-import { Box } from "grommet";
-import { Grommet } from "grommet-icons";
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { MapContainer, TileLayer } from 'react-leaflet';
+import { Box, Text } from 'grommet';
+import { Grommet } from 'grommet-icons';
 
-import GrommetMarker from "./GrommetMarker";
-import MarkerClusterGroup from "./MarkerClusterGroup";
-import { GrommetGeoJSON } from "./GrommetGeoJSON";
+import GrommetMarker from './GrommetMarker';
+import MarkerClusterGroup from './MarkerClusterGroup';
+import { GrommetGeoJSON } from './GrommetGeoJSON';
 import { Pin } from './markers';
-import { generateLocations } from "./utils/locations";
-import { geojsonLocations } from "./utils/geojson_data";
+import { generateLocations } from './utils/locations';
+import { geojsonLocations } from './utils/geojson_data';
+import { getStatusCounts, STATUS_MAP } from './utils/status';
 
 const initialStyle = {
-  height: "100%",
+  height: '100%',
+};
+
+const ClusterTip = ({ cluster }) => {
```

Comment:
Soft opinion, but I'm wondering if ClusterTip should be moved outside of the Map.jsx file?

Reply:
Good call, moved into its own file.